lib/util.js

```diff
@@ -3,26 +3,24 @@
  */
 
 /**
- * A utility function to create a verification suite.
+ * A utility function to create a key type handler.
  *
  * @param {object} options - Options hashmap.
- * @param {Function} options.generate - A method that generates a KeyPair with
- *   an optional deterministic seed.
- * @param {Function} options.from - A method that creates a verification KeyPair
+ * @param {Function} options.from - A method that creates a KeyPair
  *   from an existing serialized key.
+ * @param {Function} options.fromFingerprint - A method that creates a KeyPair
+ *   from a key fingerprint.
  *
- * @returns {object} - Verification suite.
+ * @returns {object} - Keytype handler.
  */
-export function createVerificationSuite({generate, from}) {
+export function createKeyTypeHandler({from, fromFingerprint}) {
```

Comment:
See my comments below...

```suggestion
export function createFromMultibase({from, fromFingerprint}) {
```

And then we can just return the `from` function directly here instead of returning an object.

Reply:
Addressed here [7024e9b](https://github.com/digitalbazaar/did-method-key/pull/55/commits/7024e9b680a35604ecffbc6bebeefce414a0c842)